test-jest/cartesian/Brush.spec.tsx

```diff
@@ -63,6 +70,7 @@ describe('<Brush />', () => {
     expect(screen.getAllByText(data[data.length - 1].date)).toHaveLength(1);
   });
 
+  // eslint-disable-next-line max-len
   test('mouse down on traveller will trigger the brush text display, and mouse move out will hide the brush text', () => {
```

Comment:
probably don't need to disable this line? Just let it be formatted

Reply:
I already tried. 
The line is too long to be formatted automatically.
Do I need to do something like this?

```tsx
[
  'mouse down on traveller will trigger the brush text display,',
  'and mouse move out will hide the brush text',
].join(' ')
```

Reply:
Shouldn't need to no. Strange. Feel free to keep the ignore there I guess.